### PR TITLE
Fix grammar, spelling, and consistency mistakes in bg.yml

### PR DESCRIPTION
**Pull request by ignisf**
I noticed a few annoying mistakes in the translations, here are fixes for them + some missing strings.

### CODE REVIEW

File: rails/locales/bg.yml [hunk 1]
```diff
@@ -32,40 +32,40 @@ bg:
       confirmation_instructions:
         action: "Потвърждаване на профил"
         greeting: "Добре дошли, %{recipient}!"
-        instruction: "Можете да потвърдите Вашия имейл адрес чрез долния линка:"
+        instruction: "Можете да потвърдите своя имейл адрес чрез долния линк:"
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: Might read even better as "Можете да потвърдите имейл адреса си чрез долния линк:"

File: rails/locales/bg.yml [hunk 2]
```diff
@@ -32,40 +32,40 @@ bg:
...
+        subject: "Променена парола"
       reset_password_instructions:
         action: "Смяна на парола"
-        greeting: "Здравей, %{recipient}!"
-        instruction: "Някой е поискал линк за смяна на парола. Можете да смените паролата си чрез долния линк."
-        instruction_2: "В случай, че не сте поискали смяна на паролата, моля игнорирайте настоящия имейл."
-        instruction_3: "Паролата Ви няма да бъде сменена докато не използвате горния линк, за да създадете новва."
-        subject: "Инструкции за смяна на паролата"
+        greeting: "Здравейте, %{recipient}!"
+        instruction: "Някой е поискал линк за промяна на парола Ви. Можете да промените паролата си чрез долния линк."
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: "на паролата Ви". But the old version also seems fine to me, "линк за смяна на парола".

File: rails/locales/bg.yml [hunk 3]
```diff
@@ -32,40 +32,40 @@ bg:
...
+        instruction_2: "В случай, че не Сте поискали промяна на паролата си, моля игнорирайте настоящия имейл."
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: I think the polite form is still singular, so it'd be "В случай, че не Сте поискал промяна". Not super sure about this, though, some googling about the rules might be a good idea.

File: rails/locales/bg.yml [hunk 4]
```diff
@@ -32,40 +32,40 @@ bg:
...
+        instruction_3: "Паролата Ви няма да бъде променена, докато не използвате горния линк, за да създадете нова."
+        subject: "Инструкции за промяна на парола"
       unlock_instructions:
         action: "Отключване на профил"
-        greeting: "Здравей, %{recipient}!"
-        instruction: "Натиснете върху донлия линк, за да отключите профила си:"
-        message: "Профилът Ви е заключен поради надвишаване броя на неуспешните опити за вход."
+        greeting: "Здравейte, %{recipient}!"
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: Minor typo -- the "t" is latin.

File: rails/locales/bg.yml [hunk 5]
```diff
@@ -32,40 +32,40 @@ bg:
...
+        instruction: "Кликнете върху донлия линк, за да отключите профила си:"
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: A typo that was there before as well -- "долния линк".

File: rails/locales/bg.yml [hunk 6]
```diff
@@ -32,40 +32,40 @@ bg:
...
+        message: "Профилът Ви е заключен поради надвишаване на максималния позволен брой неуспешни опити за вход."
         subject: "Инструкции за отключване"
     omniauth_callbacks:
-      failure: Не успяхме да ви оторизираме чрез %{kind}, защото "%{reason}".
-      success: "Успешно оторизиран чрез %{kind} профил."
+      failure: "Не успяхме да ви оторизираме чрез %{kind}, защото %{reason}."
+      success: "Успешно оторизация чрез %{kind} профил."
```
Comment (AndrewRadev) [inline rails/locales/bg.yml]: "Успешна оторизация"

File: rails/locales/bg.yml (hunk 3 above)
Comment (ignisf) [inline rails/locales/bg.yml]: Past passive participles should be plural when using polite form

**Comment by AndrewRadev** [comment]
:+1:

File: rails/locales/bg.yml (hunk 3 above)
Comment (mitio) [inline rails/locales/bg.yml]: A few points on this:

1. I think "Сте" should be lowercase here. It's not a pronoun, it's a verb. AFAIK only the pronouns "вас", "ви", "вие" etc. should be capitalized ([reference from 2014 in Bulgarian](https://pavlinav.wordpress.com/2014/08/20/1-70/)).
2. The comma before "че" in "В случай, че" should be removed. "В случай че" seems to be a single conjunction ([reference, again in Bulgarian](http://kaksepishe.com/zapetaya-v-sluchay-che/)).
3. What do you think about changing "моля игнорирайте" to "просто игнорирайте"?
4. What do you think about changing "Можете да промените паролата си чрез долния линк." to "Можете да я промените чрез линка по-долу."?

**Comment by ignisf** [comment]
@JasonBarnabe, I'm happy with the state of this PR. It's up for merging.

**Comment by JasonBarnabe** [comment]
Thanks!

**Pull request merged**